StreamChatClient_v3/Controllers/ChannelListControllerPublishers.swift

```diff
@@ -0,0 +1,121 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Combine
+import UIKit
+
+@available(iOS 13, *)
+extension Client {
+    /// Creates a new `ChannelListController.Publishers` object with the provided channel query.
+    ///
+    /// - Parameter query: The query specify the filter and sorting of the channels the controller should fetch.
+    /// - Returns: A new instance of `ChannelListController.Publishers`.
+    ///
+    @available(iOS 13, *)
+    public func channelListControllerPublishers(query: ChannelListQuery) -> ChannelListControllerGeneric<ExtraData>.Publishers {
+        let controller = ChannelListControllerGeneric(query: query, client: self)
+        return .init(controller: controller)
+    }
+}
+
+/// Describes possible states of Controller's remote activity.
+public enum RemoteActivity {
+    /// The controller is not listening for updates. Call `startUpdating` to fetch the remote data and start listening for changes.
+    case none
+    
+    /// Controller actively tries to refresh the local data.
+    case fetchingRemoteData
+    
+    /// Controller is listening for incoming changes.
+    case listening
+    
+    /// Remote data fetch failed. Controller might still be able to show the recent incoming changes, but it's possible
+    /// the presented data are not complete.
+    case failed(error: Error?)
+}
+
+@available(iOS 13, *)
+extension ChannelListControllerGeneric {
+    /// An object which wraps `ChannelListController` and exposes its properties as Publishers.
+    public class Publishers {
+        /// Publishes changes to the remote activity state. You can use this publisher to show activity indicator to users
+        /// if the controller is fetching the remote data.
+        public var remoteActivityPublisher: AnyPublisher<RemoteActivity, Never> {
+            remoteActivity
+                // Is something like this the way to keep `Publishers` object alive if there is a subscriber?
+                .map { [self] in _ = self; return $0 }
+                .eraseToAnyPublisher()
+        }
+        
+        /// Publishes the changes to the list of channels matching the query. You can use this publisher to animate changes
+        /// to the channel list, or to simply reload all your the data.
+        ///
+        /// Alternatively, you can use `channelsDiffableDatasourcePublisher` which provides similar functionality using the
+        /// `NSDiffableDataSource` API.
+        ///
+        /// - Note: ⚠️ The publisher publishes only **changes** to the list of channels. Use the `channels` property to load
+        /// the initial data before subscribing to the publisher.
+        public var channelChangesPublisher: AnyPublisher<[Change<ChannelModel<ExtraData>>], Never> {
```

Comment:
`ChannelListControllerPublishers.channelChangesPublisher`
I think no need to add `Publisher` suffix, it's clear that should be publishers because `ChannelListControllerPublishers`